finagle-netty4/src/main/scala/com/twitter/finagle/netty4/Netty4HashedWheelTimer.scala

```diff
@@ -24,7 +24,7 @@ private object timerTickDuration
 /**
  * A Netty timer for use with [[Netty4HashedWheelTimer]].
  */
-private class HashedWheelTimer(
+class HashedWheelTimer(
```

Comment:
can we do it while keeping this private?  or does that cause a problem?  if we can't make this private, can we make the constructor private?